SalesforceSDK/NoteSync/NoteSync.Windows/NoteSync.Windows.csproj

```diff
@@ -136,21 +136,21 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\Salesforce.SDK.App\Salesforce.SDK.App.csproj">
+    <ProjectReference Include="..\..\Salesforce.SDK.App\App.csproj">
       <Project>{b1d32173-8770-45b7-aee8-1f29af8cdc87}</Project>
-      <Name>Salesforce.SDK.App</Name>
+      <Name>App</Name>
     </ProjectReference>
-    <ProjectReference Include="..\..\Salesforce.SDK.Core\Salesforce.SDK.Core.csproj">
+    <ProjectReference Include="..\..\Salesforce.SDK.Core\Core.csproj">
```

Comment:
Salesforce.SDK.Core is the current namespace, since we have a GA release done, keeping the namesapce same